magpie/cli/create_private_key.py

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/env python3
+
+"""
+Create a private key file used to generate a JSON Web Key.
+
+This file is required when network mode is enabled in order to sign JSON Web Tokens.
+"""
+
+import argparse
+import os.path
+import sys
+from typing import TYPE_CHECKING
+
+from magpie.api.management.network.network_utils import create_private_key, pem_files
+from magpie.cli.utils import make_logging_options, setup_logger_from_options
+from magpie.constants import get_constant
+from magpie.utils import get_logger, get_settings_from_config_ini
+
+if TYPE_CHECKING:
+    from typing import Optional, Sequence
+
+    from magpie.typedefs import Str
+
+LOGGER = get_logger(__name__,
+                    message_format="%(asctime)s - %(levelname)s - %(message)s",
+                    datetime_format="%d-%b-%y %H:%M:%S", force_stdout=False)
+
+
+def make_parser():
+    # type: () -> argparse.ArgumentParser
+    parser = argparse.ArgumentParser(description="Create a private key used to generate a JSON Web Key.")
+    parser.add_argument("--config", "--ini", metavar="CONFIG", dest="ini_config",
+                        default=get_constant("MAGPIE_INI_FILE_PATH"),
+                        help="Configuration INI file to retrieve database connection settings (default: %(default)s).")
+    parser.add_argument("--key-file",
+                        help="Location to write key file to. Default is to use the first file listed in the "
```

Comment:
"*Location where to write key file*" seems more readable

Reply:
😆 yes, maybe this is a French perspective

It is the double use of `to` that sounds odd to me (though it is valid syntax). 
"*Location where the key file must be saved to*" is OK as well. 

Not a big deal though.